src/trie/trie.c

```diff
@@ -44,6 +44,13 @@ typedef struct {
 
 static void __trieNode_sortChildren(TrieNode *n);
 
+#define updateScore(n, value)                             \
+do {                                                      \
+  if (n->sortMode == Trie_Sort_Score) {                   \
```

Comment:
Why wouldn't you want to update the score anyway?

Reply:
Why should we? Branch prediction should make this free.

Reply:
But wouldn't it have effect if we switch sorting mode?

Reply:
We never switch.
A trie stay either by score or by lex.